### PR TITLE
Use local gpg when access to keyring is not needed.

### PR DESCRIPTION
**Pull request by whohoho**

### CODE REVIEW

File: gpg-client-wrapper [hunk 1]
```diff
@@ -5,10 +5,56 @@ eoo=0       # end of options reached
...
 while [[ $1 ]]; do
     if ! ((eoo)); then
         case "$1" in
+            #when those arguments are present will not use the keyring, and so they can be executed with local gpg
+            # can be used in combination with sign
+            #-c)
+            #    localgpg=1
+            #    break
+            #;;
+            --passprase-fd)
```
Comment (marmarek) [inline gpg-client-wrapper]: This alone should not trigger local gpg usage.

File: gpg-client-wrapper [hunk 2]
```diff
@@ -5,10 +5,56 @@ eoo=0       # end of options reached
 output=0    # do we try to write to file
 target=()   # where do we try to write to
 fd_for_stdout=63 # some (hopefuly) unused FD, later redirected to stdout
+localgpg=0 #use local gpg (for ex --gen-rand etc.)
+origargs="$@"
```
Comment (marmarek) [inline gpg-client-wrapper]: Use array, otherwise options with spaces will be broken.
`origargs=( "$@" )`

File: gpg-client-wrapper [hunk 3]
```diff
@@ -62,8 +111,15 @@ while [[ $1 ]]; do
     fi
 done
 
+if [ $localgpg -eq 1 ]
+then
+    exec /usr/bin/gpg $origargs
```
Comment (marmarek) [inline gpg-client-wrapper]: "${origargs[@]}"

File: gpg-client-wrapper (hunk 1 above)
Comment (whohoho) [reply gpg-client-wrapper]: removed that one, it was misspelled anyway

File: gpg-client-wrapper (hunk 2 above)
Comment (whohoho) [reply gpg-client-wrapper]: fixed it

**Comment by marmarek** [comment]
Thanks!

**Pull request merged**